sdk/dotnet/Pulumi/Testing/MockMonitor.cs

```diff
@@ -65,7 +65,9 @@ public async Task<ReadResourceResponse> ReadResourceAsync(Resource resource, Rea
             {
                 var builder = ImmutableDictionary.CreateBuilder<string, object>();
                 builder.Add("urn", urn);
-                builder.Add("id", id);
+                if (id != null) {
```

Comment:
It would be nice if you could put `{` on a new line everywhere (or set your editor to do so).

```suggestion
                if (id != null)
                {
```

Reply:
Ah, yeah--will do.